src/research.py

```diff
@@ -88,89 +115,311 @@ def get_info(self):
+onager = Weapon('Onager', 750, 550, 1.5, 50, 1, 1, 1, reload_time=700/2, ammunition=10, projectile_speed=25, siege_weapon=True, projectile_size=19)
+mortar = Weapon('Mortar', 850, 600, 1.5, 55, 1, 1, 1, reload_time=750/2, ammunition=10, projectile_speed=20, siege_weapon=True, projectile_size=17)
+cannon = Weapon('Cannon', 1000, 650, 1.5, 60, 1, 1, 1, reload_time=1000/2, ammunition=10, projectile_speed=45, siege_weapon=True, projectile_size=18)
+catapult = Weapon('Catapult', 750, 530, 1.5, 60, 1, 1, 1, reload_time=600/2, ammunition=30, projectile_speed=25, siege_weapon=True, projectile_size=19)
+
+
+all_ranged_weapons = [scorpio, ballista, onager, mortar, cannon, catapult, musket, arquebus, blunderbuss, matchlock_rifle, flintlock_rifle, flintlock_pistol, arbalest, hunting_crossbow, recurve_bow, war_bow, self_bow, hunting_bow, throwing_knives, throwing_axes, throwing_daggers, jarid, stones, darts, atlatl, sling, shortbow, longbow, javelin, bow, crossbow, sling_staff, handgun, trebuchet]
+ranged_weapon_list = [scorpio, ballista, onager, mortar, cannon, catapult, musket, arquebus, blunderbuss, matchlock_rifle, flintlock_rifle, flintlock_pistol, arbalest, hunting_crossbow, recurve_bow, war_bow, self_bow, hunting_bow, throwing_knives, throwing_axes, throwing_daggers, jarid, stones, darts, sling, atlatl, javelin, shortbow, longbow, bow, crossbow, sling_staff, handgun, trebuchet]
+
+basic_ranged_weapon_list = [hunting_crossbow, hunting_bow, throwing_knives, throwing_axes, throwing_daggers, jarid, stones, darts, sling, javelin, shortbow, bow]
 
 cloth_armor = Armor('Cloth Armor', 100, 0, 2, 0.5)
 padded_armor = Armor('Padded Armor', 200, 0, 3, 0.5)
 leather_armor = Armor('Leather Armor', 300, 0, 4, 0.4)
 wood_armor = Armor('Wood Armor', 300, 0, 5, 0.3)
 chainmail = Armor('Chaimail', 1000, 1, 8, 0.25)
-plate = Armor('Plate', 2000, 2, 12, 0.15)
+plate = Armor('Plate', 2000, 2, 12, 0.15, heavy=True)
 
-armor_list = [cloth_armor, padded_armor, leather_armor, wood_armor, chainmail, plate]
+#Armor - Kenny Additions
+gambeson = Armor('Gambeson', 600, 1, 5, 0.35)
+hauberk = Armor('Hauberk', 1300, 2, 9, 0.30)
+lamellar = Armor('Lamellar', 1700, 2, 10, 0.25, heavy=True)
 
-armor_list = [wood_armor, chainmail, plate]
-basic_armor_list = [cloth_armor, leather_armor, wood_armor, padded_armor]
+tunic = Armor('Tunic', 80, 0, 2, 0.45)
+robe = Armor('Robe', 50, 0, 1, 1)
+fur_coat = Armor('Fur Coat', 10, 0, 1, 0.5)
+jerkin = Armor('Jerkin', 100, 0, 3, 0.25)
+linen_shirt = Armor('Linen', 20, 0, 0, 0.75)
+aketon = Armor('Aketon', 70, 0, 2.5, 0.35)
+tabard = Armor('Tabard', 60, 0, 1.5, 1.5)
+ragged_armor = Armor('Ragged Armor', 10, 0, 0, 0.5)
+tribal_wrappings = Armor('Tribal Wrappings', 5, 0, 0, 0.25)
+
+lorica = Armor('Lorica', 500, 2, 7, 0.50)
+byrnie = Armor('Byrnie', 600, 1.7, 8, 0.75, heavy=True)
+
+desert_mail = Armor('Desert Mail', 1000, 1, 6, 1)
```

Comment:
desert mail?

Reply:
I don't know... I probably look up the correct term later